api-service/src/routes/Router.ts

```diff
@@ -20,6 +21,8 @@ router.patch("/v1/datasets/update", setApiId("api.datasets.update"), onRequest({
 router.get("/v1/datasets/read/:dataset_id", setApiId("api.datasets.read"), onRequest({ entity: Entity.Management }), DatasetRead)
 router.post("/v1/datasets/list", setApiId("api.datasets.list"), onRequest({ entity: Entity.Management }), DatasetList)
 router.get('/v1/data/exhaust/:datasetId', setApiId("api.data.exhaust"), onRequest({ entity: Entity.Management }), dataExhaust);
+router.post('/v1/template/create/:templateName', setApiId("query.template.create"), onRequest({ entity: Entity.Management }), createQueryTemplate);
```

Comment:
why template name is in params ?
It should be part of request body

Reply:
resolved